vms/platformvm/state/state.go

```diff
@@ -1287,10 +1284,12 @@ func (s *state) loadPendingValidators() error {
 				return err
 			}
 
-			validator := s.pendingStakers.getOrCreateValidator(staker.SubnetID, staker.NodeID)
-			validator.validator = staker
+			pendingValidator := s.pendingStakers.getOrCreateValidator(staker.SubnetID, staker.NodeID)
```

Comment:
q: do we care sorting pending validators as well? I don't think we ever iterate though them. GetValidatorsSet only care about current validators, right?